source/developers/how-tos/enable-translations-new-repo.rst

```diff
@@ -183,35 +177,41 @@ Testing translations sync in your fork
 Before submitting a pull request for review in the `openedx-translations repo`_, you should test the workflow
 on a fork by following the steps below:
 
-#. Fork the `openedx-translations repo`_.
-#. Make a pull request to your fork and modify the `extract-translation-source-files.yml`_ workflow to use your
-   repo, your organization name and the branch in the clone section.
-   For example, the `frontend-lib-special-exams testing pull request`_ uses the ``Zeit-Labs/frontend-lib-special-exams`` repository with the branch
-   set via ``ref: fix-i18n``.
+#. Add the ``make extract_translation`` into your fork of the new repository e.g. ``Zeit-Labs/credentials`` in a new branch e.g. ``omar/add-pull-translations``
+#. Fork the `openedx-translations repo`_ e.g. ``Zeit-Labs/openedx-translations``.
+#. In your fork, modify the `extract-translation-source-files.yml`_ file in a new branch e.g. ``omar/add-credentials``.
+#. Go to the ``Actions`` tab in your repository (e.g. ``Zeit-Labs/openedx-translations``)
+#. From the left section, pick the `"Extract Translation Source Files" section in your fork`_
+#. Click on the "Run workflow" dropdown button with the following parameters:
 
-#. Modify the `extract-translation-source-files.yml`_ workflow to run ``pull_request`` events.
+   - **Use workflow from branch:** ``omar/add-credentials``
```

Comment:
Is this a brand that people will have, or should this be like,

```suggestion
   - **Use workflow from your branch:** ``your-branch-name``
```

Also at line 189